src/main/java/seedu/address/model/person/Subject.java

```diff
@@ -102,7 +102,7 @@ public static String getValidSubjectMessage() {
      */
     public static boolean isValidSubjectName(String subjectName) {
         requireNonNull(subjectName);
-        return inEnum(subjectName.toUpperCase(), Subjects.class);
+        return inEnum(subjectName, Subjects.class);
```

Comment:
I think this line should still have the .toUpperCase() so that "math" would return true and it will be reformatted in constructor

Reply:
inenum is case insensitive so no need